spider_bot.py

```diff
@@ -1,46 +1,50 @@
-from irc import *
-from secrets import *
-import os
-import random
-import time
+from irc import IRC
+from secrets import get_server, get_channel
+from time import sleep
+
 
 channel = get_channel()
 server = get_server()
 nickname = "spider_bot"
 
-irc = IRC()
-irc.connect(server, channel, nickname)
+
+irc_obj = IRC()
+irc_obj.connect(server, channel, nickname)
 
 
-while 1:
-    text = irc.get_text()
+# Infinite loop until disconnected
+while True:
+    text = irc_obj.get_text()
     print text
 
     if "PRIVMSG" in text and channel in text and "*snap*" in text.lower():
-        irc.send(channel, "Alexis...I don't feel so good...")
-        irc.quit("disintegrated")
+        irc_obj.send(channel, "Alexis...I don't feel so good...")
+        irc_obj.quit("disintegrated")
 
         if nickname == "spider_bot":
             nickname = "CAPTAIN_MARVEL"
         elif nickname == "CAPTAIN_MARVEL":
             nickname = "spider_bot"
 
-        time.sleep(5)
+        sleep(5)
 
-        irc = IRC()
-        irc.connect(server, channel, nickname)
+        irc_obj = IRC()
+        irc_obj.connect(server, channel, nickname)
 
     if "PRIVMSG" in text and channel in text and "goat" in text.lower():
-        irc.send(channel, "THE HERD")
+        irc_obj.send(channel, "THE HERD")
 
-    if "PRIVMSG" in text and channel in text and ("tom brady" in text.lower() or "tb12" in text.lower() or "touchdown tommy" in text.lower()):
-        irc.send(channel, "the GOAT")
+    if "PRIVMSG" in text and channel in text and ("tom brady" in text.lower()
+                                                  or "tb12" in text.lower() or
+                                                  "touchdown tommy" in
+                                                  text.lower()):
```

Comment:
We could and should make this cleaner by structuring the conditionals in a more readable way. For example:
```
if conditional 
   and conditional
   and (conditional
       or conditional
       or conditional)
   and conditional:
    commands...
```
Not as extreme as this but within reason to make it more easily readable

Reply:
Done